src/main/Main.cpp

```diff
@@ -197,7 +198,7 @@ try {
     auto backend = data::make_Backend(config);
 
     // Manages clients subscribed to streams
-    auto subscriptions = feed::SubscriptionManager::make_SubscriptionManager(config, backend);
+    auto subscriptions = std::make_shared<feed::SubscriptionManagerRunner>(config, backend);
```

Comment:
For now , both work, because this main never ends. But i am thinking maybe we can control these service/managers lifespan more precious by shared_ptr in future?

Reply:
Maybe we don't need to do it now? Because reading this in the future we may have a question why it is shared_ptr. It is always easy to change it here to unique_ptr/shared_ptr later if we need more lifetime control.